drake/solvers/SNOPTWrapper.cpp

```diff
@@ -0,0 +1,343 @@
+
```

Comment:
This might be a good time to try the new SNOPT version. The interface of SNOPT is changed.